samples/cellular/nrf_provisioning/README.rst

```diff
@@ -115,13 +115,33 @@ The following files are available:
 * :file:`prj.conf` - Standard default configuration file.
 * :file:`overlay-coap.conf` - Enables CoAP transfer protocol support.
 * :file:`overlay-at_shell.conf` - Enables writing of large certificates from AT shell.
-* :file:`overlay-large_cert.conf` - Adjusts buffer sizes to handle nRF Cloud certificates.
 
 Building and running
 ********************
 
 .. |sample path| replace:: :file:`samples/cellular/nrf_provisioning`
 
+.. note::
+
+   This service is only compatible with nRF91x1 devices.
+
```

Comment:
You might want to have an intro sentence here before the tabs. Something like:

To build the sample, you can use either HTTP or CoAP protocol.